db/001-public.sql

```diff
@@ -49,7 +49,6 @@ CREATE TABLE IF NOT EXISTS public.data (
     CONSTRAINT fk_data_timeseries FOREIGN KEY (timeseries) REFERENCES public.timeseries
 ) PARTITION BY RANGE (obstime);
 CREATE INDEX IF NOT EXISTS data_timestamp_index ON public.data (obstime);
-CREATE INDEX IF NOT EXISTS data_timeseries_index ON public.data USING HASH (timeseries);
```

Comment:
Since we know there's now an implicit index created by the unique constraint (which we want), I wonder if we should change the unique constraint to a primary key? It has the same effect, but I think it makes the intention behind the schema more obvious

Reply:
That's a good point, I changed all unique constraints to primary keys!